### PR TITLE
restrict QuestionSearch to past 2 years

### PR DESCRIPTION
**Pull request by LeoMcA**
https://github.com/mozilla/sumo-project/issues/740

### CODE REVIEW

File: kitsune/search/v2/search.py [hunk 1]
```diff
@@ -55,7 +55,13 @@ def get_highlight_fields(self):
 
     def get_filter(self):
         filters = [
+            # restrict to the question index
             DSLQ("term", _index=self.get_index()),
+            # only return questions created in the last two years
+            DSLQ(
+                "range",
+                question_created={"gte": datetime.now(timezone.utc) - timedelta(days=365.25 * 2)},
```
Comment (akatsoulas) [inline kitsune/search/v2/search.py:63]: let's use a variable for the two years. also why float?

Comment (LeoMcA) [reply kitsune/search/v2/search.py:63]: It's the mean number of days in a year :P

Comment (akatsoulas) [reply kitsune/search/v2/search.py]: cool, let's make it an integer. floor or ceil totally up to you

File: kitsune/search/v2/search.py [hunk 2]
```diff
@@ -8,6 +8,8 @@
 from kitsune.search.v2.base import SumoSearch
 from kitsune.sumo.urlresolvers import reverse
 
+QUESTION_SEARCH_DELTA = timedelta(days=365.25 * 2)
```
Comment (akatsoulas) [inline kitsune/search/v2/search.py:11]: Let's keep only the days here and move the timedelta function into DSLQ query

**Review by akatsoulas — APPROVED** [review]
r+wc (squash bfr merging)

**Pull request merged**